src/app/AppContainer.js

```diff
@@ -2,55 +2,109 @@
  * @format */
 
 import { connect } from 'react-redux';
-import {
-	updateBlockAttributes,
-	focusBlockAction,
-	moveBlockUpAction,
-	moveBlockDownAction,
-	deleteBlockAction,
-	createBlockAction,
-	parseBlocksAction,
-	serializeToNativeAction,
-	mergeBlocksAction,
-} from '../store/actions';
+import { isEqual } from 'lodash';
+
 import MainApp from './MainApp';
+import { parse, serialize } from '@wordpress/blocks';
+import { withDispatch, withSelect } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+import RNReactNativeGutenbergBridge from 'react-native-gutenberg-bridge';
+
+const mapStateToProps = ( state, ownProps ) => {
+	let blocks = state.blocks;
+	let refresh = !! ownProps.refresh;
+
+	const newBlocks = ownProps.blocks.map( ( block ) => {
```

Comment:
This is a hack to set the `focus` property on each block. Once we use the `isSelected()` selector in our components we can simplify this